README.md

```diff
@@ -129,7 +129,7 @@ Found 2 dependencies with mismatching versions across the workspace. Fix with `-
 
 ## References
 
-* [npm workspaces](https://docs.npmjs.com/cli/v8/using-npm/workspaces)
+* [npm workspaces](https://docs.npmjs.com/cli/v9/using-npm/workspaces)
```

Comment:
It looks like we can omit the version number so we don't have to keep updating this:

```suggestion
* [npm workspaces](https://docs.npmjs.com/cli/using-npm/workspaces)
```